src/pages/ShoppingCart.page.js

```diff
@@ -24,4 +39,75 @@ export class ShoppingCartPage extends BaseSwagLabPage {
     async removeCartItemById(id) {
         await this.cartItems.nth(id).locator(this.removeItemSelector).click();
     }
+
+    //verify cart page
+    async verifyCartPage(){
+        await this.page.waitForURL('**/cart.html')
+        await expect(this.page.getByText('Your Cart')).toBeVisible();
+    }
+
+    //check selected item in cart
+    async checkSelectedItem(itemArray){
+        await expect(this.page.locator('.cart_item').first().locator('.inventory_item_name')).toHaveText(itemArray[0].name);
+        await expect(this.page.locator('.cart_item').nth(1).locator('.inventory_item_name')).toHaveText(itemArray[1].name);
+
+        await expect(this.page.locator('.cart_item').first().locator('.inventory_item_desc')).toHaveText(itemArray[0].description);
+        await expect(this.page.locator('.cart_item').nth(1).locator('.inventory_item_desc')).toHaveText(itemArray[1].description);
+
+        await expect(this.page.locator('.cart_item').first().locator('data-test=inventory-item-price')).toHaveText('$'+(itemArray[0].price).toString());
+        await expect(this.page.locator('.cart_item').nth(1).locator('data-test=inventory-item-price')).toHaveText('$'+(itemArray[1].price).toString());
+    }
+    async goToCheckout(){
+        await this.page.getByRole('button', { name: 'Checkout' }).click();
+    }
+
+    async goToFinish(){
+        await this.page.getByRole('button', { name: 'Finish' }).click();
+    }
+    async goToHome(){
+        await this.page.getByRole('button', { name: 'Back Home' }).click();
+    }
+    
+    //verify Checkout page
+    async verifyCheckoutPage(){
+        await this.page.waitForURL('**/checkout-step-one.html')
+        await expect(this.page).toHaveURL(/.*checkout-step-one/);
+        await expect(this.page.getByText('Checkout: Your Information')).toBeVisible();
+    }
+
+    async performCheckout(firstName, lastName, zip) {
+        await this.firstNameInput.fill(firstName);
+        await this.lastNameInput.fill(lastName);
+        await this.zipInput.fill(zip);
+        await this.continueButton.click();
+    }
+
+    async itemTotalPrice(productArray){
+        let itemsTotalPrice = 0 
+        for (let i=0; i < productArray.length; i++) {
+            itemsTotalPrice += productArray[i].price
+        }
+        return itemsTotalPrice
+    }
+
+    async taxCalculate(productArray){
+        let tax = await this.itemTotalPrice(productArray) * 0.08
```

Comment:
Why do we use let instead of const? 
We'd better to move 0.08 to const and name it proper, so we could know what does it mean.

Reply:
@hlazkov  this const variable = 0.08 should stay in this function?